ObjCVoiceQuickstart/ViewController.m

```diff
@@ -284,11 +284,11 @@ - (void)outgoingCall:(TVOOutgoingCall *)outgoingCall didFailWithError:(NSError *
 
 #pragma mark - AVAudioSession
 - (void)routeAudioToSpeaker {
-    NSError *error = nil;
-    if (![[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryPlayAndRecord
-                                          withOptions:AVAudioSessionCategoryOptionDefaultToSpeaker
-                                                error:&error]) {
-        NSLog(@"Unable to reroute audio: %@", [error localizedDescription]);
+    NSError * error;
+    [[AVAudioSession sharedInstance] overrideOutputAudioPort:kAudioSessionOverrideAudioRoute_Speaker
```

Comment:
We should check that the return value of the method is `NO` and then log out the error.